### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by drewpotter**
Opening a PR so that @spencerldixon can see what I have done so far and make comments where necessary on the spanish translations that have been added so far.

### CODE REVIEW

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -70,6 +70,6 @@ def set_question
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def question_params
-      params.require(:question).permit(:text, :x_weight, :y_weight)
+      params.require(:question).permit(:x_weight, :y_weight, :text)
```
Comment (spencerldixon) [inline app/controllers/questions_controller.rb]: Why

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
     <% end %>
 
-    <div class="collapse navbar-collapse navbar-toggleable-sm" id="collapsingNavbar">
+    <div class="collapse navbar-collapse navbar-toggleable-sm justify-content-between" id="collapsingNavbar">
       <ul class="nav navbar-nav">
-        <%= active_link_to "Home", root_path %>
-        <%= active_link_to "Take the Survey", new_survey_path %>
+        <%= active_link_to t('menu.home'), root_path %>
+        <%= active_link_to t("menu.take_the_survey"), new_survey_path %>
         <%= active_link_to "About the Project", about_the_project_path %>
```
Comment (spencerldixon) [inline app/views/layouts/_navigation.html.erb]: These should also be translated

File: app/views/questions/show.html.erb [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
+  <p>
+  <strong>Text (<%= t('language', locale: translation.locale) %>)</strong>
+  <%= translation.text %>
+  </p>
+<% end %>
 
 <p>
-  <strong>X Weight:</strong>
-  <%= @question.x_weight %>
+<strong>X Weight:</strong>
```
Comment (spencerldixon) [inline app/views/questions/show.html.erb]: Restore indentation

File: app/controllers/questions_controller.rb (hunk 1 above)
Comment (drewpotter) [reply app/controllers/questions_controller.rb]: Uhm @spencerldixon I didn't intentionally change this file, I thought I just merged with master but maybe there was a conflict and I merged it wrongly, I will restore it back.

File: app/views/layouts/_navigation.html.erb (hunk 2 above)
Comment (drewpotter) [reply app/views/layouts/_navigation.html.erb]: Ok @spencerldixon I will add them to the yml files.

File: app/views/questions/show.html.erb (hunk 3 above)
Comment (drewpotter) [reply app/views/questions/show.html.erb]: Sure @spencerldixon I will restore the indentation here.

File: app/views/survey/demographic_questions/_experience_in_other_fields.html.erb [hunk 4]
```diff
@@ -1,12 +1,12 @@
 <p class="card-text">
   <%= radio_button_tag "demographics[#{question.short_name}]", 'Yes' %>
-  <%= label_tag 'Yes' %>
+  <%= label_tag t('questionnaire.yes') %>
```
Comment (spencerldixon) [inline app/views/survey/demographic_questions/_experience_in_other_fields.html.erb:3]: Is this fixed now @drewpotter ?

Comment (drewpotter) [reply app/views/survey/demographic_questions/_experience_in_other_fields.html.erb:3]: Yes it is fine now @spencerldixon

**Pull request merged**